README.md

```diff
@@ -29,3 +29,5 @@ SuiteCloud CLI for Node.js is an open source project. Pull Requests are currentl
 
 ## [License](./LICENSE.txt)
 Copyright (c) 2021 Oracle and/or its affiliates The Universal Permissive License (UPL), Version 1.0.
+
+By installing the NPM package, you are accepting the installation of a dependency. See [License](/https://www.oracle.com/downloads/licenses/oracle-free-license.html) for details.
```

Comment:
What dependency?
I think we should say something like: 
By installing the NPM package, you are accepting the installation of a the SuiteCloud SDK dependency under this license.
I would use this kind of same formula in the other readme files.

Reply:
Hi @Montsant19, can you please help with this?

Reply:
Hi, @antoniofruci , @davidecorreu makes sense to me: By installing the NPM package, you are accepting the installation of the SuiteCloud SDK dependency under this license.

Reply:
I am going to change it in all the readme files accordingly then. Thank you @Montsant19